data/tutorials/language/0lg_00_basic_datatypes.md

```diff
@@ -0,0 +1,807 @@
+
+<!-- It is also an expression-oriented language: everything is a value and every value has a type. Functions and types are the two foundational principles of OCaml. The OCaml type system is highly expressive, providing many advanced constructs while being easy to use and unobtrusive. Thanks to type inference, programs can be written without type annotations, except for documentation purposes and a few corner cases. The basic types and the type combination operations enable a vast range of possibilities.
+
+> Type annotation means that it's necessary to write the value type to tell the compiler what to do, as opposed to having the compiler infer the type.
+-->
+
+This tutorial will cover atomic types, such as integers and Booleans, before moving on to predefined compound types, like strings and lists. It ends with a section about user-defined types, namely variants and records.
+
+Before proceeding, it's necessary to have completed the [Get Started](https://ocaml.org/docs/get-started) series of tutorials as well as [Functions and Values](/docs/values-and-functions).
+
+<!--
+The goal of this tutorial is to provide for the following capabilities:
+- Handle data of all predefined types using dedicated syntax
+- Write variant type definitions: simple, recursive, and polymorphic
+- Write record type definitions (without mutable fields)
+- Write type aliases
+- Use pattern matching to define functions for all basic type
+-->
+
+## Predefined Types
+
+### Integers, Floats, Booleans, and Characters
+
+#### Integers
+
+The `int` type is the default and basic integer type in OCaml. When you enter a whole number, OCaml recognises it as an integer, as shown in this example:
+```ocaml
+# 42;;
+- : int = 42
+```
+
+The `int` type represents platform-dependent signed integers. This means `int` does not always have same the number of bits. It depends on underlying platform characteristics, such as processor architecture or operating system. Operations on `int` values are provided by the [`Stdlib`](/api/Stdlib.html) and the [`Int`](/api/Int.html) modules.
+
+Usually, `int` has 31 bits in 32-bit architectures and 63 in 64-bit architectures because one bit is reserved for OCaml's runtime operation. The standard library also provides [`Int32`](/api/Int32.html) and [`Int64`](/api/Int64.html) modules, which support platform independent operations on 32- and 64-bit signed integers. These modules are not detailed in this tutorial.
```

Comment:
Can part (or all) of the paragraph be deleted since it talks about things not covered in this tutorial?